tests_for_ranges.py

```diff
@@ -0,0 +1,14 @@
+from detect_ranges_with_count import detect_ranges
+from check_valid_input import check_valid
+from print_reading_ranges import print_reading_ranges
+
+assert(detect_ranges([3, 3, 5, 4, 10, 11, 12]) == [(3,5,4),(10,12,3)])
+assert(detect_ranges([3, 3]) == [(3,3,2)])
+assert(detect_ranges([4,5]) == [(4,5,2)]) #failing case
+
+assert(check_valid("2,3,4,5,6,6,7,8") == True)
+assert(check_valid([2,3,4,5,6,6,7,8]) == False)
+
+assert(print_reading_ranges([(3,5,4),(10,12,3)]) == True)

```

Comment:
this is a weak test case which just checks execution and not the actual printing. How do we know that the method is printing exactly how we want it to ?